[esx_addons]/esx_lscustom/client/main.lua

```diff
@@ -237,6 +237,13 @@ function GetAction(data)
                         modType = k,
                         modNum = false
                     })
+                elseif v.modType == 23 then
+                    table.insert(elements, {
+                        label = " " .. TranslateCap('by_default'),
+                        modType = "modFrontWheels",
+                        modNum = -1,
+                        wheelType = -1
```

Comment:
set additional value
`price = 1`
or something

Reply:
Hi, thanks for the review.

I put the price to 1 like you said, but I'm thinking of maybe setting the default price in the config so anyone can just change it like the other wheels prices, what do you think?

Reply:
> 

config is the proper way to do it 👍

Reply:
Ok it's pushed, i've set it to 2 by default since the other wheels are at 4+ price, thought 50% would be a good value but if you prefer 1 i change it in a sec